src/main/java/uk/gov/ons/ctp/integration/rhsvc/event/impl/CaseEventReceiverImpl.java

```diff
@@ -52,7 +52,7 @@ public void acceptCaseEvent(CaseEvent caseEvent) throws CTPException {
     if (surveyUpdateOpt.isPresent()) {
       SurveyUpdate surveyUpdate = surveyUpdateOpt.get();
       try {
-        if (surveyUpdate.getName().equalsIgnoreCase("SIS")) {
+        if (!surveyUpdate.getSampleDefinitionUrl().endsWith("social.json")) {
```

Comment:
please put the postive logic first.  ie reverse the if / else logic so we deal with SOCIAL before we deal with SIS.
Also we need unit test for code coverage , as discussed on slack , and mentioned in a previous PR review.